GameOfLifeApp/GameLogic.cs

```diff
@@ -0,0 +1,160 @@
+        {
+            // Instantiate random number generator
+            Random rnd = new Random();
+
+            for (int row = 0; row < CountOfRows; row++)
+            {
+                for (int column = 0; column < CountOfColumns; column++)
+                {
+                    CurrentField[row, column] = rnd.Next(0, 2) == 0;
+                }
+            }
+        }
+
+        /// <summary>
+        /// populates the field with Glider the Shape. 
+        /// </summary>
+        public void CreateGliderShape()
+        {
+            CurrentField[0, 0]= true;
+            CurrentField[2, 0] = true;
+            CurrentField[1, 1] = true;
+            CurrentField[2, 1] = true;
+            CurrentField[1, 2] = true;
+        }
+
+        /// <summary>
+        /// Gets number of alive cells around a given cell. 
+        /// </summary>
+        /// <param name="currentRow">Row of a current cell.</param>
+        /// <param name="currentColumn">Column of a current cell.</param>
+        /// <returns>Return number of alive neighbours.</returns>
+        private int NeighboursCount(int currentRow, int currentColumn)
+        {
+            int count = 0;
+            for (int row = currentRow - 1; row <= currentRow + 1; row++)
+            {
+                for (int column = currentColumn - 1; column <= currentColumn +1; column++)
+                {
+                    int actualRow;
+                    int actualColumn;
+                    if (row < 0)
+                    {
+                        actualRow = CurrentField.GetLength(0) - 1;
+                    }
+                    else if (row == CurrentField.GetLength(0))
+                    {
+                        actualRow = 0;
+                    }
+                    else
+                    {
+                        actualRow = row;
+                    }
+                    if (column < 0)
+                    {
+                        actualColumn = CurrentField.GetLength(1) - 1;
+                    }
+                    else if (column == CurrentField.GetLength(1))
+                    {
+                        actualColumn = 0;
+                    }
+                    else
+                    {
+                        actualColumn = column;
+                    }
+                    count += CurrentField[actualRow, actualColumn] ? 1 : 0;
+                }
+            }
+            count -= CurrentField[currentRow, currentColumn] ? 1 : 0;
+
+            return count;
+        }
+
+        /// <summary>
+        /// Gets new boolean values according to statements.
+        /// </summary>
+        /// <param name="neighboursCount">Number of alive cells around a given cell.</param>
+        /// <param name="isCurrentAlive">Given cell is alive. Value of isCurrentAlive: true.</param>
+        /// <returns>Return the next value for a given cell.</returns>
+        private static bool GetCellOffsprings(int neighboursCount, bool isCurrentAlive)
```

Comment:
In this case, you dont need the static keyword, as this method will be accessible within this class even without it to the other methods.

Reply:
Removed.